envs/example/allinone-rhel/group_vars/all.yml

```diff
@@ -0,0 +1,97 @@
+---
+openstack_install_method: 'distro'
+openstack_distro_type: 'osp'
+
+stack_env: example_allinone
+floating_ip: 172.16.0.100
+xtradb:
+  sst_auth_password: asdf
+validate_certs: false
+percona:
+  replication: False
+
+rhn_subscription:
```

Comment:
should variable this so that subscription information can be passed as --extra-vars

rhn_subscription:
  username: "{{ rhn_user }}" 
  password: "{{ rhn_pass }}"